tests/Rule/IsFloatTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace Particle\Validator\Tests\Rule;
+
+use Particle\Validator\Rule\IsFloat;
+use Particle\Validator\Validator;
+
+class IsFloatTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var Validator
+     */
+    protected $validator;
+
+    protected function setUp()
+    {
+        $this->validator = new Validator();
+    }
+
+    public function testReturnsTrueOnValidFloat()
+    {
+        $value = 3.14;
+
+        $this->validator->required('value')->float();
+
+        $result = $this->validator->validate([
+            'value' => $value,
+        ]);
+
+        $this->assertTrue($result->isValid());
+    }
+
+    /**
+     * @dataProvider getInvalidFloatValues
+     *
+     * @param mixed $value
+     */
+    public function testReturnsFalseOnInvalidFloat($value)
+    {
+        $this->validator->required('value')->float();
+
+        $result = $this->validator->validate([
+            'value' => $value,
+        ]);
+
+        $this->assertFalse($result->isValid());
+
+        $expected = [
+            'value' => [
+                IsFloat::NOT_A_FLOAT => $this->getMessage(IsFloat::NOT_A_FLOAT),
+            ],
+        ];
+
+        $this->assertEquals($expected, $result->getMessages());
+    }
+
+    public function getInvalidFloatValues()
+    {
+        return [
+            ['foo'],
+            [9000],
+            [true],
```

Comment:
could you also verify that `'1,5'` also isn't a float? This because it happens quite often that Dutch people insert values with a `,` instead of a `.`, which a system can't handle.